test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -75,10 +75,13 @@ public void setUp() throws Exception {
 
     @Rule
     public ExpectedException thrown = ExpectedException.none();
+    
 
     @Test
     public void addPerson_emptyAddressBook() throws Exception {
-        emptyAddressBook.addPerson(bobChaplin);
+        emptyAddressBook.addPerson(bobChaplin); 
+        Person bobChaplin_confirmPerson = emptyAddressBook.getAllPersons().iterator().next();
```

Comment:
This naming style (using `_` in variables), is not consistent with the rest of the code base.